Documentation meta-descriptions review N° 4 (#5558)
wazuh/wazuh-documentation: source/user-manual/reference/tools/agent-upgrade.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: The agent_upgrade program allows you to list outdated agents and upgrade them. Learn more about it in this section of the Wazuh documentation.

Reviewer: ```suggestion
  :description: List outdated agents and upgrade them using the agent_upgrade program. Learn more about it in this section of the Wazuh documentation.
```
